bin/dbatools-buildref-index.json

```diff
@@ -4181,7 +4181,7 @@
         {
             "Version": "15.0.2000",
             "SP": "RTM",
-            "SupportedUntil": "2025-01-07"
+            "SupportedUntil": "2030-01-08"

```

Comment:
This date should be reverted. We do not track the extended support date, only the mainstream.